### DISCUSSION TITLE
A question about the init-model mode

### DISCUSSION THREAD

**Discussion by TerryHangLyu** [deepmodeling/deepmd-kit#1528] [Q&A] [answered]
Dear developers and users,
   I have a question when browzing the Zhihu introduction of DeepMD-kit. That's is how can we know that the prossibility for the old data set and new data set is 0.1 and 0.9 respectively from the keywords. 
  From this part ("auto_prob_style":  "prob_sys_size; 0:3:9; 3:5:1"), I can not see how the prossibility for the old data set and new data set is defined.
![image](https://user-images.githubusercontent.com/55949849/156632897-8403eca8-b761-4f24-ba83-2e00c852f276.png)

**Comment by njzjz** [comment]
> That's is how can we know that the prossibility for the old data set and new data set is 0.1 and 0.9 respectively from the keywords.

The probability is __0.9__ and __0.1__, respectively, instead of 0.1 and 0.9.

**Reply by TerryHangLyu** [reply]
Thank you!

**Comment by likefallwind** [comment, marked as answer]
You can check this：https://docs.deepmodeling.org/projects/deepmd/en/master/train/training-advanced.html

![image](https://user-images.githubusercontent.com/10484087/156687103-3df13dff-a973-4fb2-b74a-92cd603cb674.png)

**Reply by TerryHangLyu** [reply]
Thank you!

**Comment by likefallwind** [comment]
We also add some explaination in Chinese：https://zhuanlan.zhihu.com/p/348671837?
